controllers/userController.js

```diff
@@ -156,21 +157,49 @@ const deleteUserAddresses = async (req, res) => {
 
 const forgotPassword = async (req, res) => {
   const { username } = req.body
+
   try {
     const userData = await userModel.findOne({ where: { username } })
     if (userData) {
       const passwordResetToken = jwt.sign({
         data: `${userData.username} ${userData.password}`
       }, process.env.jwtSecKey, { expiresIn: 60 * 10 });
       await userModel.update({ passwordResetToken: passwordResetToken }, { where: { username } })
-      res.status(200).send(passwordResetToken)
 
+      const transporter = nodemailer.createTransport({
```

Comment:
move this code to a separate file, which will be use to do only email related stuff. @ashok-choudhary-0